src/logs.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+const debug = require('debug')
+
+class Logger {
```

Comment:
You're using this class in several modules; want to make it into its own module?

Reply:
Sounds good to me :+1: